components/04-views/dashboard/account-info/account-info.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const checkboxPassword = document.querySelector('input#changePassword'),
+      checkboxEmail    = document.querySelector('input#changeEmail'),
+      confirmPassword  = document.querySelector('.confirmPassword'),
+      email            = document.querySelector('.email'),
+      fields           = document.querySelector('.dashboard-form__fields'),
+      newPassword      = document.querySelector('.newPassword'),
+      password         = document.querySelector('.password'),
+      title            = document.querySelector('.dashboard-form__title-form');
+
+checkboxEmail.addEventListener('change', () => {
+  if (checkboxPassword.checked && checkboxEmail.checked) {
+    title.innerHTML = 'Change Email and Password';
+
+    email.classList.remove('dashboard-form--hidden');
+    password.classList.remove('dashboard-form--hidden');
+    newPassword.classList.remove('dashboard-form--hidden');
+    confirmPassword.classList.remove('dashboard-form--hidden');
```

Comment:
really? You have to toggle 4 classes? to make it work? 
and if you use the same class name so many time, please move it to variable. Try to limit this elements. We use BEM blocks, so maybe we can use one modifier to change the state of show/hide those elements? Refactor please

You repeat a lot of code. It you really, really need to toggle 4 classes, write a function for it. Make some function to bind listeners for elements. 
To refactor

Reply:
Changed to the function with parameter and with using `if` condition

Reply:
better now. Please add some comment inside this file that is only for preview inside components and for now there is a script in Magento Dashbord which can handle it by default